site-src/geps/gep-713.md

```diff
@@ -38,10 +48,116 @@ portable.
 * Define all potential policies that may be attached to resources
 * Design the full structure and configuration of policies
 
+## Background and concepts
+
+One of the things we’ve found when making Gateway API is that we often need to be
+able to represent information _about_ objects _outside of_ those objects. That is,
+sometimes we need ways to be able to affect how an object is interpreted in the
+API, without representing the description of those effects inside the spec of the
+object.
+
+Sometimes, this is because we can’t change the spec of the object to hold the
+information we need ( ReferenceGrant, from GEP-709, is an example), and sometimes
+it’s because we want the information we need to flow across multiple objects
+(this is what Policy Attachment is for).
```

Comment:
This might be better as the opening statement, e.g. "Sometimes we can change spec ... e.t.c." and then lead into "and so sometimes we need to affect how an object is interpreted without changing the spec"?

Reply:
I changed the wording of these first two paragraphs around a little. PTAL.